root.go

```diff
@@ -50,6 +51,7 @@ func init() {
 	rootCmd.PersistentFlags().BoolVarP(&debug, "debug", "d", false, "Enable the debug")
 
 	rootCmd.AddCommand(checkCmd)
+	checkCmd.PersistentFlags().BoolVarP(&checkGinRoutes, "gin-routes", "g", false, "Check the endpoint patterns on a gin router")
```

Comment:
My suggestion would be to use the short flag `-t` to test the configuration against a real Gin router. E.g:
```
checkCmd.PersistentFlags().BoolVarP(&testGinRoutes, "test-gin-routes", "t", false, "Test the endpoint patterns against a real gin router on selected port")
```